Makefile

```diff
@@ -0,0 +1,19 @@
+# Build an executable for running go server after database migrations pass
```

Comment:
https://gist.github.com/Stratus3D/a5be23866810735d7413

https://gist.github.com/dnishimura/2961173

https://ariejan.net/2015/10/03/a-makefile-for-golang-cli-tools/

http://www.slideshare.net/RaPz1/build-golang-projects-properly-with-makefiles